packages/cli/src/metrics/prometheus-metrics.service.ts

```diff
@@ -145,6 +146,37 @@ export class PrometheusMetricsService {
 		);
 	}
 
+	/**
+	 * Set up metric for user activity based on backend calls
+	 */
+	private initActivityMetric(app: express.Application) {
+		const activityGauge = new promClient.Gauge({
+			name: this.prefix + 'last_activity',
+			help: 'last user activity (backend call).',
+			labelNames: ['timestamp'],
+		});
+
+		activityGauge.set({ timestamp: new Date().toISOString() }, 1);
+
+		app.use(
```

Comment:
we are already setting up a metric collection middleware for these exact same routes in `initRouteMetrics`. Could we combine these two, instead of adding another middleware to the stack?